sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -487,14 +509,20 @@ def to_open_api(self) -> OpenApiModel:
 @dataclass
 class StandardRouterEnsemblerConfig(RouterEnsemblerConfig):
     def __init__(
-        self, experiment_mappings: List[Dict[str, str]], fallback_response_route_id: str
+        self,
+        fallback_response_route_id: str,
+        experiment_mappings: List[Dict[str, str]] = None,
+        route_name_path: str = "",
     ):
         """
         Method to create a new standard ensembler
 
-        :param experiment_mappings: configured mappings between routes and treatments
+        :param route_name_path: configured routh name path that points to the route name within a given treatment config
+                :param experiment_mappings: configured mappings between routes and treatments
```

Comment:
nit: formatting is misaligned

Reply:
Thanks for spotting that! Strange black didn't fix that 🤔